src/scenario.py

```diff
@@ -188,7 +193,192 @@ def run(self, strategy_name, options):
+            json_results["avg flex per window"] = {
+                "04-10": avg_flex_per_window[0],
+                "10-16": avg_flex_per_window[1],
+                "16-22": avg_flex_per_window[2],
+                "22-04": avg_flex_per_window[3],
+                "unit": "kW",
+                "info": "Average Flex-power range per time window"
+            }
+
+            # sum of used energy per window
+            sum_energy_per_window = [sum([t[1] for t in w]) / ts_per_hour for w in load_window]
+            json_results["sum of energy per window"] = {
+                "04-10": sum_energy_per_window[0],
+                "10-16": sum_energy_per_window[1],
+                "16-22": sum_energy_per_window[2],
+                "22-04": sum_energy_per_window[3],
+                "unit": "kWh",
+                "info": "Total drawn energy per time window"
+            }
+
+            # avg standing time
+            # don't use info from flex band, as standing times might be interleaved
+            # remove last empty standing count
+            for counts in load_count:
+                if counts[-1] == 0:
+                    counts = counts[:-1]
+            num_loads = sum(map(len, load_count))
+            if num_loads > 0:
+                avg_stand_time = sum(map(sum, load_count)) / ts_per_hour / num_loads
+            else:
+                avg_stand_time = 0
+            # avg total standing time
+            # count per car: list(zip(*count_window))
+            total_standing = sum(map(sum, count_window))
+            avg_total_standing_time = total_standing / len(self.constants.vehicles) / ts_per_hour
+            json_results["avg standing time"] = {
+                "single": avg_stand_time,
+                "total": avg_total_standing_time,
+                "unit": "h",
+                "info": "Average duration of a single charge event and total time spent charging"
+            }
+
+            # percent of standing time in time window
+            perc_stand_window = list(map(
+                lambda x: x/total_standing if total_standing > 0 else 0,
+                map(sum, count_window)))
+            json_results["standing per window"] = {
+                "04-10": perc_stand_window[0],
+                "10-16": perc_stand_window[1],
+                "16-22": perc_stand_window[2],
+                "22-04": perc_stand_window[3],
+                "unit": None,
+                "info": "Share of standing time per time window"
+            }
+
+            # avg needed energy per standing period
+            intervals = flex["intervals"]
+            avg_needed_energy = sum([i["needed"] for i in intervals]) / len(intervals)
+            json_results["avg needed energy"] = {
+                "value": avg_needed_energy,
+                "unit": "kWh",
+                "info": "Average amount of energy needed to reach the desired SoC"
+                        " (averaged over all vehicles and charge events)"
```

Comment:
Bisher sieht es immer gut aus. Nur einfach der Hinweis bitte nochmal zu prüfen, dass "needed" auch in Energie (kWh) angegeben ist.